src/playwright-runner.ts

```diff
@@ -419,56 +425,65 @@ async function runPlaywright(nodeBin, runCfg) {
   }
   procArgs.push(...suite.testMatch);
 
-  args = _.defaultsDeep(suite, args);
-  if (args.testIgnore && args.testIgnore.length > 0) {
-    process.env.TEST_IGNORE = args.testIgnore;
+  // args = _.defaultsDeep(suite, args);
```

Comment:
🤔

Reply:
I don't think this call to `defaultsDeep` is necessary. So this is merging `suite` with `args`, but the only time `args.testIgnore` is accessed is immediately after the merging...It's easier to understand without the merging and just operating directly on `suite.testIgnore`, imo.